modules/serverless-ossm-v2x-jwt.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// * serverless/networking/serverless-ossm-jwt.adoc
+
+[id="serverless-ossm-v2x-jwt_{context}"]
+= Using JSON Web Token authentication with {ProductShortName} 2.x and {ServerlessProductName}
+
+.Procedure
+
+. Create the `RequestAuthentication` resource in a serverless application namespace, that is a member in the `ServiceMeshMemberRoll` object,  by copying the following code into a YAML file:
```

Comment:
Maybe we should add here an _"each"_ word. So, it would be clear that this needs to be done for each namespace they would like to integrate.

```suggestion
. Create the `RequestAuthentication` resource in each serverless application namespace, that is a member in the `ServiceMeshMemberRoll` object,  by copying the following code into a YAML file:
```